subsys/usb/class/Kconfig.cdc

```diff
@@ -1,15 +1,26 @@
 # Copyright (c) 2016 Wind River Systems, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-menuconfig USB_CDC_ACM
-	bool "USB CDC ACM Device Class support"
+DT_COMPAT_ZEPHYR_CDC_ACM_UART := zephyr,cdc-acm-uart
+
+menu "USB CDC ACM Class support"
+
+config USB_CDC_ACM_HAS_DT_COMPAT
+	bool
+	default $(dt_compat_enabled,$(DT_COMPAT_ZEPHYR_CDC_ACM_UART))
+	imply USB_CDC_ACM
+	help
+	  Helper symbol to select USB_CDC_ACM if compatible node is enabled.
+
+config USB_CDC_ACM
+	bool "Enable USB CDC ACM Class support"
 	select SERIAL_HAS_DRIVER
 	select SERIAL_SUPPORT_INTERRUPT
 	select RING_BUFFER
 	select UART_INTERRUPT_DRIVEN
 	help
-	  USB CDC ACM device class support.
-	  Default device name is "CDC_ACM_0".
+	  Enable USB CDC ACM Class support.
```

Comment:
Every class has something like `USB <class_name> (Device) Class support` except for CDC ACM right now.

Reply:
I ment: remove `"Enable"`